node/node.go

```diff
@@ -510,6 +515,12 @@ func (n *Node) SendTransaction(
 		Signature: signature,
 	}
 
+	txCopy := tx
+	err = n.validateTx(txCopy, senderShardId)
```

Comment:
no need for copy. Send through pointer.

Reply:
It was needed for tx signature check. but that part was removed as the signature verification is already done by the interceptor